gateway/ln-gateway/src/client.rs

```diff
@@ -95,8 +95,9 @@ impl GatewayClientBuilder {
                 .open(root_secret)
                 .await
         } else {
-            let client_config =
-                fedimint_api_client::download_from_invite_code(&invite_code).await?;
+            let client_config = fedimint_api_client::api::net::Connector::default()
+                .download_from_invite_code(&invite_code)
```

Comment:
mention todo: don't use default connector here

Reply:
@maan2003 Do you think we should already address this in this PR, or just keep as TODO ?

Reply:
just leave it as todo